lib/assets/Html.js

```diff
@@ -214,9 +213,25 @@ extendWithGettersAndSetters(Html.prototype, {
                     throw err;
                 }
             }
-            if (isEmpty) {
-                // Remove the sole text node caused by the text === '' hack above:
-                this._parseTree.removeChild(this._parseTree.firstChild);
+            if (isFragment) {
+                this._parseTree = document.body;
+                // Install an assortment of bound methods from the document object on the parse tree:
+                [
+                    'createElement', 'createElementNS',
+                    'createTextNode', 'createComment', 'createDocumentFragment', 'createProcessingInstruction', 'importNode', 'adoptNode',
+                    'createAttribute', 'createAttributeNS',
+                    'createEvent',
+                    'createTreeWalker',
+                    'createNodeIterator',
+                    'getElementById',
+                    'getElementsByName'
```

Comment:
Not too much if we leave out the properties that already exist on `document.body`.

Here's the complete list:

```js
> var doc = require('jsdom').jsdom('<!DOCTYPE html><html><body></body></html>'); for (var p in doc) {if (!(p in doc.body)) {console.log(p, typeof doc[p]);}}
location object
getElementById function
createElement function
createElementNS function
createDocumentFragment function
createTextNode function
createComment function
createProcessingInstruction function
importNode function
adoptNode function
createAttribute function
createAttributeNS function
createEvent function
createTreeWalker function
getElementsByName function
open function
close function
write function
writeln function
implementation object
URL string
documentURI string
origin string
compatMode string
characterSet string
charset string
inputEncoding string
contentType string
doctype object
documentElement object
referrer string
cookie string
lastModified string
readyState string
body object
head object
images object
links object
forms object
currentScript object
defaultView object
activeElement object
onreadystatechange undefined
anchors object
applets object
styleSheets object
visibilityState string
onvisibilitychange undefined
createExpression function
createNSResolver function
evaluate function
createNodeIterator function
```

Some of them don't make a lot of sense to expose (eg. `head`), but I don't know if it would actually harm anything.